pages/applications/index.vue

```diff
@@ -1,98 +1,123 @@
+              <a href="/public/VF_Application_Form.docx">{{
+                documents.applicationlink
+              }}</a>
+              {{ documents.applicationlabelB }}</label
+            >
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="project-budget"
+              >{{ documents.budgetlabel
+              }}<a href="/public/VF_Budget_Template_Form.xls">{{
+                documents.budgetlink
+              }}</a></label
+            >
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="current-budget">{{ documents.currentbudget }}</label>
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="current-budget">{{ documents.irsletter }}</label>
+          </li>
+        </ul>
     </fieldset>
-    <p class="page-description">
-      Our applications follow the guidelines of the Common Grant Application & Budget Template. You can <a href="/public/CGA_User_Guide.pdf">download the CGA User Guide here</a>.
-    </p>
-    <h3>Three Options to Submit Application</h3>
-    <h4 style="text-align: center;">1. Submit Application Online</h4>
-    <p class="page-description">You can now submit your application online!</p>
-    <a class="action-button" href="/applications/new-application">
-      Submit Application Online
-    </a>
-    <h5 style="text-align: center;">OR</h5>
-    <h4 style="text-align: center;">2. Email Application</h4>
-    <p class="page-description">You can simply email the documents to <a href="mailto:info@vatterottfoundation.org">info@vatterottfoundation.org</a></p>
-    <h5 style="text-align: center;">OR</h5>
-    <h4 style="text-align: center;">3. Mail-In Application</h4>
-    <p class="page-description">
-      Please send application materials to:
+
+      <div style="text-align: center;">
+        {{ guidelines.applicationguidelines }}
+        <a href="/public/CGA_User_Guide.pdf"></br>{{ guidelines.guidelinelink }}</a>
+      </div>
+      <br />
+
+      <h2>{{ options.header3 }}</h2>
+
+      <div style="text-align: center;">
+       <p class="page-description"> 
+         <h4><strong>{{ options.online.option1 }}</strong></h4> 
+         </br>{{ options.online.option1text }}
+      </p>
+        <a class="action-button" href="/applications/new-application">
+          {{ options.online.submitbutton }}
+        </a>
+        <strong>{{ options.or }}</strong>
+        <p class="page-description">
+          <h4><strong>{{ options.email.option2 }}</strong></h4>
+          <br /> 
+          {{ options.email.option2text }} 
+          <a href="mailto:info@vatterottfoundation.org">{{ options.email.emailaddresslink }}</a>
+        </p>
+      <br />
+      <strong>{{ options.or2 }}</strong><br />
+        <p class="page-description">
+          <h4><strong>{{ options.mail.option3 }}</strong></h4>
+          <br />
+         {{ options.mail.option3text }}
+        </p>
```

Comment:
Let's make this options area a markdown field instead

Reply:
Last night you said we could leave this area be... please confirm that